src/engine.h

```diff
@@ -74,16 +86,21 @@ typedef struct {
 } tetris_stats_t;
 
 typedef struct {
-    int w_height, w_width;
-    SDL_Window *window;
-    SDL_Renderer *renderer;
-    int target_framerate;
-    tetris_event_t event_stack[EVENT_STACK_SIZE];
-    int event_stack_top;
-    tetris_board_t board;
-    double last_frame_duration;
-    tetris_stats_t stats;
-    TTF_Font *font;
+	int w_height, w_width;
+	SDL_Window* window;
+	SDL_Renderer* renderer;
+	int target_framerate;
+	tetris_state_t game_state;
+	tetris_event_t event_stack[EVENT_STACK_SIZE];
+	int event_stack_top;
+	tetris_board_t board;
+	double last_frame_duration;
+	double last_delta_time;
```

Comment:
there's two properties that do more or less the same thing. maybe one of them should go away

Reply:
That's intentional. The first measures rendering time, exclusively, the second one also includes measurements of the time spent between frames. They have similar names but ultimately measure different things.